appsec/src/helper/remote_config/client_handler.cpp

```diff
@@ -47,10 +47,11 @@ client_handler::ptr client_handler::from_settings(service_identifier &&id,
     }
 
     std::vector<remote_config::listener_base::shared_ptr> listeners = {};
-    if (dynamic_enablement) {
+    if (dynamic_enablement || eng_settings.schema_extraction.enabled) {
```

Comment:
Perhaps we could add the listener in all cases, the method `client::compute_client_status()` already takes into account whether the RC value should be used or not.

Reply:
I rather leave it like this and not populate those products we know we are not going to use